src/__tests__/test-service.ts

```diff
@@ -108,12 +108,12 @@ export class TestTigrisService {
 			reply.setStatus("created");
 			callback(undefined, reply);
 		},
-		// eslint-disable-next-line @typescript-eslint/no-empty-function
+		/* eslint-disable @typescript-eslint/no-empty-function */
 		createOrUpdateCollection(
 			call: ServerUnaryCall<CreateOrUpdateCollectionRequest, CreateOrUpdateCollectionResponse>,
 			callback: sendUnaryData<CreateOrUpdateCollectionResponse>
 		): void {},
-		// eslint-disable-next-line @typescript-eslint/no-empty-function
+		/* eslint-enable @typescript-eslint/no-empty-function */
```

Comment:
rever this.

Reply:
This is by choice. I disabled rule for a couple of line and want to enable it for the rest of the lines